### DISCUSSION TITLE
使用Clash无法访问解析国外域名

### DISCUSSION THREAD

**Discussion by PikachuCN** [vernesong/OpenClash#3767] [Q&A]
用Passwall可以，但是用OpenClash，我在本机测试Ping或者Nslookup均无法解析国外域名，如ip.sb等，关闭后又正常，导致一直好像用不了，不知道要怎么排查，Config.yaml文件在Andorid手机也正常。节点也是正常。不知道哪里配置有问题呢？

**Comment by ghost** [comment]


**Reply by PikachuCN** [reply]
恩，Ping不通主要显示不了域名，Nslookup显示解析域名失败，好像是他默认使用路由ip做解析，无法解析。
那天折腾了一下passwall，突然OpenClash又一切正常了。但是现在很奇怪，ping或者nslookup都会显示 192.18.0.x，无论域名是否存在解析这种奇怪地址。

**Reply by PikachuCN** [reply]
运行几次又开始无法解析域名，使用curl 会提示：
rooteiStore0S:/#curlip.sb
curl:(52)Empty reply from server

日志会显示curl: (28) SSL connection timeout curl: (28) SSL connection timeout curl: (28) SSL connection timeout


现在我启动办法是，先启动一次passwall，然后关掉，启动openclash就正常，如果我关掉openclash，再重新打开就不正常。

这问题有点让我莫名其妙。。

passwall性能好像挺慢，同一个节点，passwall跑30M，openclash跑300M

**Reply by ghost** [reply]
把能运行和不能运行的调试日志都上传上来才可能有人看
> 调试日志在插件设置-调试日志中生成，并非只有运行日志，如调试日志过长，可作为附件上传
> **隐私提示: 上传此日志前请注意检查、屏蔽公网IP、节点、密码等相关敏感信息**
